### ISSUE TITLE
Keep saying couldnt open file for reading etc

### ISSUE THREAD

**Issue by LordRavenI** [Fate-Grand-Automata/FGA#839]
https://user-images.githubusercontent.com/75866234/130308684-164f31f8-63cd-476b-9e3f-8af2205ec453.mp4

### Preparation

- [X] I tested the [latest release](https://github.com/Fate-Grand-Automata/FGA/releases/tag/canary)
- [X] I looked at other issues (even the [closed ones](https://github.com/Fate-Grand-Automata/FGA/issues?q=is%3Aissue+is%3Aclosed))
- [X] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)

### FGO server

NA

### FGA build number

1480

### Describe the bug

When i try to use the app it keeps saying either no skadi, no teatime, etc. And when i went to the files where these should be and i allow access it didnt have anything. Is this a me problem or app problem? And hoe should i fox it if its a me problem

### Video

https://user-images.githubusercontent.com/75866234/130307248-33fc7bd3-7480-400d-b678-9ced5aa628c1.mp4

### Device model

Honor 9x

### Android version

10

### Screen size

2340 x 1080

### RAM

6gb

**Comment by MathewSachin** [comment]
More options/storage/extract default support images

**Comment by LordRavenI** [comment]
> More options/storage/extract default support images

I tried this and it still didnt work. Then i tried delated the files and reinstalling the app but it still cant find the images

**Comment by reconman** [comment]
So which folder did you select and what happens if you extract the images?

**Comment by LordRavenI** [comment]
> So which folder did you select and what happens if you extract the images?
> 
> Also, is your phone storage full?

I did allow support folder, then try the other folders one by one when it didnt work. Its not full, though i guess i can delate some apps just to make sure

**Comment by LordRavenI** [comment]
> So which folder did you select and what happens if you extract the images?
> supports, etc, everything one by one and it didnt work
> Also, is your phone storage full?
Nope, just cleared it
2nd time writing since i dont see the eariler one, if its the same sorry
Jist tried delated even the support folder and the fgo auto app files. Everything. And try to redownload...still dont have it...

**Comment by LordRavenI** [comment]
But now when i try i cant even find the support folder...

**Comment by LordRavenI** [comment]
After dealteing everything and redownloading plus restarting and time and some diigging i was able to finally find thr supports and this time get it to work as it shouldve

**Comment by weakedsick03** [comment]
![image](https://user-images.githubusercontent.com/23069430/140518246-67563212-579a-4f1e-84fe-2c796622229e.png)

Hello. App always stops whenever I run a command that selects a CE. I tried extracting Default Support images but it says "Couldn't extract support images". Any idea how to circumvent this? 

I think this has to do with Bluestacks being stingy with folder access but I was able to import my commands anyway. What's the deal? 
![image](https://user-images.githubusercontent.com/23069430/140518150-4c7098c6-5367-40b2-a87c-881de168b949.png)
